backend/cmd/server/transcription.go

```diff
@@ -112,3 +116,84 @@ func (t TranscriptionAPI) GetPreview(ctx context.Context, req *connect.Request[a
 
 	return connect.NewResponse(&apiv1.Preview{Url: preview}), nil
 }
+
+func (t TranscriptionAPI) SubmitTranscription(ctx context.Context, req *connect.Request[apiv1.SubmitTranscriptionRequest]) (*connect.Response[apiv1.Void], error) {
+	fmt.Printf("Received transcription for VXID %s: %+v\n", req.Msg.GetVXID(), req.Msg.GetTranscription())
+
+	// Trigger flow
+	queue := getQueue()
+	workflowOptions := client.StartWorkflowOptions{
+		TaskQueue: queue,
+	}
+
+	wfRun, err := t.temporalClient.ExecuteWorkflow(ctx, workflowOptions, ingestworkflows.ImportSubtitles, ingestworkflows.ImportSubtitlesInput{
+		VXID:      req.Msg.VXID,
+		Subtitles: mapApiTranscriptionToModel(req.Msg.Transcription),
+		Language:  "no", // Hardcoded to norwegian for now
```

Comment:
What is the consequence of this?

Reply:
None really at this point because we don't support multiple Json subtitle files.